recipes/binutils/binutils.inc

```diff
@@ -129,9 +130,6 @@ RDEPENDS_${PN}-strip = "libbfd"
 RDEPENDS_${PN}-ar = "libbfd"
 RDEPENDS_${PN}-strings = "libbfd"
 
-# The binutils package is used to pull in all provided util features
-RDEPENDS_${PN} = "${AUTO_PACKAGE_UTILS_PROVIDES}"
```

Comment:
You mentioned earlier that this line was redundant.  How is it that?  How is all the auto-package-utils packages pulled in by RDEPENDS_${PN} without this line?